gui/default/syncthing/folder/editIgnoresModalView.html

```diff
@@ -14,12 +14,14 @@
     </dl>
   </div>
   <div class="modal-footer">
-    <div class="pull-left"><span translate>Editing</span> <code>{{currentFolder.path}}{{system.pathSeparator}}.stignore</code></div>
```

Comment:
Yeah I think we should disable ignore button until the form is valid. Which reduces the spectrum of messages